Eastern-Research-Group/mywaterway
# feature/package-upgrades

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3376896 - I noticed there were a few other packages that needed minor upgrades, so I figured I would go ahead and take care of those and create a separate PR for the ones with breaking changes.

## Main Changes:
* Upgrade packages without breaking changes.

## Steps To Test:
1. Run `npm ci` to get the correct packages.
2. Go through the app and make sure everything still works. 

## TODO:
- [ ] Upgrade packages with breaking changes (highcharts and react-table).
